TKGame/BackEnd/EntityManager.cs

```diff
@@ -80,7 +80,7 @@ public static void Update(GameTime gameTime)
             foreach (var entity in entities)
             {
                 entity.Update(gameTime);
-                entity.Collide(currentStage.StageWalls);
+                entity.Collide(currentStage.StageBlocks);
```

Comment:
What is the purpose of changing this?

Reply:
Walls and Spikes are basically the same thing. The only difference is that we will potentially want to be able to differentiate them. Therefore Blocks are just walls and they have a type to them such as Wall or Spikes. This way we can add moving platforms or doors that are in essence walls that just have slightly difference update functions but don't need to be called separately
i.e. collide.stage.walls, collide.stage.spikes, collide.stage.platforms etc